pkg/objectstore/s3/s3.go

```diff
@@ -14,8 +15,16 @@ import (
 )
 
 func getSession(backupLocation *stork_api.BackupLocation) (*session.Session, error) {
+	// AWS SDK fetches the correct endpoint based on region provided if endpoint is passed empty
+	// For all AWS endpoint, we can pass in empty
+	var endpoint string
+	if strings.Contains(backupLocation.Location.S3Config.Endpoint, "amazonaws.com") {
```

Comment:
Only do this if the endpoint is exactly "s3.amazonaws.com"

Reply:
Fixed.